.github/workflows/publish.yml

```diff
@@ -0,0 +1,14 @@
+jobs:
+  pypi-publish:
+    name: Upload release to PyPI
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/pywebsocket3
+    permissions:
+      id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
+    steps:
+    # retrieve your distributions here
+
+    - name: Publish package distributions to PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
I guess `release/v1` needs to be replaced with an actual tag in our repository? It should probably match the version number in `setup.py`.

Reply:
I think this version is to specify the version for github action `pypa/gh-action-pypi-publish`, so I don't think it's for pywebsocket's version